packages/deployment/app/config.yml

```diff
@@ -134,6 +145,9 @@ default:
           messageLimit: 10
           state: ENABLED
 
+    kinesisFailure:
```

Comment:
This is the sqs def for the fallback lambda DLQ.  Should this be a standard Lambda DLQ?  Or do we create one for each time we set up a lambda DLQ?  what about the name?

Reply:
I think it would be more maintainable/cleaner to have a standard lambda DLQ, similar to how we have a single startSF SQS queue. I think that would make processing items from the queue a bit simpler as well, since we wouldn't have to focus on what queue we're reading from, just the content of the entry.

That being said, I don't know if we have sufficient information in the DQL messages to distinguish between the failed lambda instances that created them...